client/voter/README.md

```diff
@@ -0,0 +1,301 @@
+  "leaf_index": 1,
+  "proposed_governor": "0x64cfb1fa0fa7da3da0097fee9d9ab546b9bcd029",
+  "sibling_path_nodes": [
+    [
+      58,
+      206,
+      177,
+      20,
+      129,
+      246,
+      70,
+      127,
+      216,
+      86,
+      7,
+      155,
+      215,
+      201,
+      120,
+      64,
+      16,
+      3,
+      241,
+      90,
+      165,
+      89,
+      116,
+      128,
+      126,
+      19,
+      141,
+      88,
+      172,
+      146,
+      157,
+      247
+    ],
+    [
+      153,
+      51,
+      253,
+      54,
+      18,
+      29,
+      214,
+      212,
+      25,
+      250,
+      168,
+      14,
+      248,
+      246,
+      211,
+      40,
+      152,
+      227,
+      223,
+      136,
+      243,
+      43,
+      110,
+      16,
+      216,
+      46,
+      68,
+      52,
+      160,
+      15,
+      128,
+      0
+    ]
+  ]
+}
+Please Sign the following using your `wdkg` ECDSA key:
+Encoded Vote: 000000000000000000000000000000000000000000000000000000000000000100000000000000000000000064cfb1fa0fa7da3da0097fee9d9ab546b9bcd02
+```
+
+You should then sign the encoded vote value using your `wdkg` ECDSA Key then save the signature.
+
+4. Create the vote transaction:
+
+```bash
+Cast votes to change the governor
+
+Usage: tangle-voter cast --vote <VOTE> --signature <SIGNATURE>
+
+Options:
+      --vote <VOTE>
+      --signature <SIGNATURE>
+  -h, --help                   Print help
+```
+
+* Example:
+
+```bash
+./target/release/tangle-voter cast \
+    --vote 000000000000000000000000000000000000000000000000000000000000000100000000000000000000000064cfb1fa0fa7da3da0097fee9d9ab546b9bcd029000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000023aceb11481f6467fd856079bd7c978401003f15aa55974807e138d58ac929df79933fd36121dd6d419faa80ef8f6d32898e3df88f32b6e10d82e4434a00f8000 \
+    --signature 0x3aceb11481f6467fd856079bd7c978401003f15aa55974807e138d58ac929df7599933fd36121dd6d419faa80ef8f6d32898e3df88f32b6e10d82e4434a00f80 \
+    --vote 000000000000000000000000000000000000000000000000000000000000000100000000000000000000000064cfb1fa0fa7da3da0097fee9d9ab546b9bcd029000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000023aceb11481f6467fd856079bd7c978401003f15aa55974807e138d58ac929df79933fd36121dd6d419faa80ef8f6d32898e3df88f32b6e10d82e4434a00f8000 \
+    --signature 0x3aceb11481f6467fd856079bd7c978401003f15aa55974807e138d58ac929df7599933fd36121dd6d419faa80ef8f6d32898e3df88f32b6e10d82e4434a00f80
+```
+
+Finally it will print the transaction that you can use [`cast`](https://book.getfoundry.sh/cast/) to send it to the signature bridge smart contract.
+
+```
+   46,
```

Comment:
This output looks a bit sloppy. Do we need these random bytes really?

Reply:
Updated.